Sources/Brave/Frontend/Browser/BrowserViewController/BrowserViewController+Menu.swift

```diff
@@ -295,6 +295,12 @@ extension BrowserViewController {
               }
             }
           }
+          
+          MenuItemButton(icon: Image(braveSystemName: "leo.warning.triangle-outline"), title: Strings.Shields.reportABrokenSite) {
+            browserViewController.dismiss(animated: true) {
+              browserViewController.showSubmitReportView(for: tabURL)
+            }
+          }
```

Comment:
Make an activity for this and put this in the list of activities so it shows up in the share sheet as well (then delete this since it will be implicitly here)

Reply:
I see, ok, done